custom/ewsghana/utils.py

```diff
@@ -100,5 +100,9 @@ def get_reporting_types(domain):
     ]
 
 
+def can_receive_email(user, verified_number):
+    return user.email and verified_number.backend_id and verified_number.backend_id == 'MOBILE_BACKEND_TWILIO'
```

Comment:
You may want to consider checking for != 'MOBILE_BACKEND_TEST' as I mentioned in my previous comment so that you can support testing with other real gateways besides Twilio, but I won't block the pr on it.